webapp/src/components/content/checkboxElement.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react'
+import {injectIntl, IntlShape} from 'react-intl'
+
+import {MutableCheckboxBlock} from '../../blocks/checkboxBlock'
+import {IContentBlock} from '../../blocks/contentBlock'
+import CheckIcon from '../../widgets/icons/check'
+import mutator from '../../mutator'
+import {Editable} from '../editable'
+
+import {contentRegistry} from './contentRegistry'
+import './checkboxElement.scss'
+
+type Props = {
+    block: IContentBlock
+    readonly: boolean
+    intl: IntlShape
+}
+
+type State = {
+    active: boolean
+    title: string
+}
+
+class CheckboxElement extends React.PureComponent<Props, State> {
+    constructor(props: Props) {
+        super(props)
+        this.state = {active: Boolean(props.block.fields.value), title: props.block.title}
```

Comment:
Recommend adding a field accessor to checkboxBlock to get the "value" state as a bool.